src/olympia/addons/tests/test_views.py

```diff
@@ -1704,6 +1704,8 @@ def test_get_addon_not_found(self):
 
 
 class TestAddonViewSetDetail(AddonAndVersionViewSetDetailMixin, TestCase):
+    client_class = APITestClient
```

Comment:
Hmm, can't we set that in `TestCase` or introduce a new `APITestCase` if we're going to use it mostly everywhere anyway? Maybe it's good the way it is though but it's one of those things people (including me) love to forget to add if creating new code.